cliconfig/file.go

```diff
@@ -120,19 +120,16 @@ func parseLine(line string) (key string, value string, err error) {
 	}
 
 	if len(splitString) != 2 {
-		err = errors.New("Can't separate key from value")
-		return
+		return "", "", fmt.Errorf("can't separate key from value in string %q, no valid separators (= or :) found", line)
```

Comment:
i love my two lil guys 
<img width="85" alt="CleanShot 2023-02-02 at 11 04 22@2x" src="https://user-images.githubusercontent.com/15753101/216173857-f58b8c76-05b2-43c2-86ad-bdb6031a38f3.png">

just vibing. hanging out

Reply:
Love it!